terraform/testnet/testnet/templates/loadtest.yaml

```diff
@@ -30,7 +30,7 @@ spec:
             - --emit-tx
             - --mint-file=/opt/aptos/etc/mint.key
             - --chain-id={{ .Values.genesis.chain_id | default .Values.genesis.era }}
-            {{- if .Values.validatorLite }}
```

Comment:
Can you explain why we need this? Is it to get around mem-pool slowness?

Reply:
Yeah to get around state sync slowness from validators to fullnodes.. 

just one less factor to consider when enabled.